src/Prism.Core/Mvvm/ErrorsContainer.cs

```diff
@@ -12,7 +12,14 @@ namespace Prism.Mvvm
     public class ErrorsContainer<T>
     {
         private static readonly T[] noErrors = new T[0];
+        /// <summary>
+        /// Delegate to be called when raiseErrorsChanged is invoked
+        /// </summary>
         protected readonly Action<string> raiseErrorsChanged;
+
+        /// <summary>
+        /// KeyValuePairs of the string error and a List of the types of the error object(s)
```

Comment:
This shouldn't reference the KeyValuePairs, it should reference the `<see cref="Dictionary{string, List{T}}" />`

Reply:
I have replaced KeyValuePairs with <see cref="Dictionary{string, List{T}}" />